libraries/c_sdk/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -661,7 +676,10 @@ static IotMqttError_t _deserializeIncomingPacket( _mqttConnection_t * pMqttConne
             }
             else
             {
-                EMPTY_ELSE_MARKER;
+                /* Deserialization of PINGRESP using MQTT Lightweight API returns one of the following status
+                 * #MQTTBadParameter , #MQTTBadResponse and # MQTTSuccess, so combining both the bad status codes to IOT_MQTT_BAD_RESPONSE
```

Comment:
"Any error in deserializing a packet needs to be considered as a bad response." How about that comment?

Reply:
Yeah sounds good